Server/CBXCUITestServer.m

```diff
@@ -79,8 +88,17 @@ + (void)start {
 - (void)start {
     NSError *error;
     BOOL serverStarted = NO;
-
-    [self.server setPort:CBX_DEFAULT_SERVER_PORT];
+    
+    NSString *portNumberString = [CBXCUITestServer valueFromArguments: NSProcessInfo.processInfo.arguments
```

Comment:
Instead of using a key/value pair for arguments, I would prefer using NSProcessInfo.processInfo.environment.

Reply:
done